src/sentry/data_export/tasks.py

```diff
@@ -52,6 +52,8 @@ def assemble_download(
     offset=0,
     bytes_written=0,
     environment_id=None,
+    retries=3,
+    count_down=60,
```

Comment:
Should this be `countdown` instead? also should we update `default_retry_delay` in the decorator to match?
https://docs.celeryproject.org/en/v4.4.7/userguide/calling.html?highlight=apply_async#eta-and-countdown

Reply:
good catch fixed in 8ee91de